aptos-move/aptos-aggregator/src/delta_math.rs

```diff
@@ -310,6 +310,19 @@ impl DeltaHistory {
             max_underflow_negative_delta: new_max_underflow,
         })
     }
+
+    pub fn stricter_than(&self, other: &DeltaHistory) -> bool {
+        self.max_achieved_positive_delta >= other.max_achieved_positive_delta
```

Comment:
Should this be < instead of >=?

Reply:
values that satisfy try_add(5)=true are stricter than values that satisfy try_add(3)=true

i.e. with limit 10, 6 doesn't satisfy first , but satisfies second

so larger max_achieved_positive_delta is stricter.